1.3.0 breaks on install

I have prism-react-renderer as dependency of my Docusaurus project and upon upgrading this package it breaks at `yarn install`:

```
error /app/node_modules/prism-react-renderer: Command failed.
Exit code: 127
Command: patch-package
Arguments:
Directory: /app/node_modules/prism-react-renderer
Output:
/bin/sh: patch-package: not found
```

My team is seeing a similar error when installing via `npm install`:

```
> prism-react-renderer@1.3.0 postinstall /Users/eh012472/Code/terra-framework/node_modules/prism-react-renderer
> patch-package

sh: patch-package: command not found
npm WARN @cerner/package-json-lint-config-terra@1.2.0 requires a peer of @cerner/package-json-lint@^0.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @octokit/plugin-request-log@1.0.4 requires a peer of @octokit/core@>=3 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-formatjs@2.21.0 requires a peer of eslint@8 but none is installed. You must install peer dependencies yourself.

npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! prism-react-renderer@1.3.0 postinstall: `patch-package`
npm ERR! spawn ENOENT
```

--- a/package.json
+++ b/package.json
@@ -15,11 +15,11 @@
     "themes"
   ],
   "scripts": {
+    "prebuild": "patch-package",
     "build": "rollup -c rollup.config.js",
     "test": "jest",
     "flow": "flow check",
     "format": "prettier --write 'src/**/*.js'",
-    "postinstall": "patch-package",
     "prepublishOnly": "run-p flow build"
   },
   "babel": {

